api/pom.xml

```diff
@@ -9,7 +9,7 @@
     <parent>
         <artifactId>treasury-parent</artifactId>
         <groupId>me.lokka30</groupId>
-        <version>1.0.1-SNAPSHOT</version>
+        <version>1.1-SNAPSHOT</version>
```

Comment:
`1.1.0-SNAPSHOT`

Reply:
@lokka30 yannick did it correctly. 1.1-SNAPSHOT

Reply:
Are you certain? The API version is formatted `major.minor`, the plugin version is formatted `major.minor.patch. Unless this value is meant to be the API version which I doubt.

Reply:
oh wait that's the plugin version. the plugin version shall be 1.0.1-SNAPSHOT. we don't change anything in the plugin. this change shall be reverted